.zombienet-tests/0001-checkSync.ts

```diff
@@ -0,0 +1,20 @@
+import { connect } from "./utils"
+
+export async function run(_nodeName: string, networkInfo: any) {
+  const { chainHead } = await connect(networkInfo)
+
+  let count = 0
+
+  await new Promise((resolve, reject) => {
+    const chainHeadFollower = chainHead(
+      true,
+      (event) => {
+        if (event.type === "finalized" && ++count === 2) {
+          resolve(chainHeadFollower.unfollow())
+        }
+      },
+      reject,
+    )
+  })
+  return count
```

Comment:
```suggestion
  return await new Promise((resolve, reject) => {
    const chainHeadFollower = chainHead(
      true,
      (event) => {
        if (event.type === "finalized" && ++count === 2) {
          chainHeadFollower.unfollow()
          resolve(count)
        }
      },
      reject,
    )
  })
```

Reply:
```
bob: ts-script ./0001-checkSync.ts within 140 seconds
```
Should assert on the `count` value?

Reply:
Thank you for that but not really. That is a remainer that should not be here. Since the resolving happens in the time limit given, there is no need to check against the (known) `count`.

Reply:
so there is no need to return `count` in the function